src/Autofac.Integration.ServiceFabric/AutofacActorExtensions.cs

```diff
@@ -71,5 +71,16 @@ public static class AutofacActorExtensions
 
             return registration;
         }
+
+        public static IRegistrationBuilder<TActorService, ConcreteReflectionActivatorData, SingleRegistrationStyle>
+            RegisterActorService<TActorService>(
+                this ContainerBuilder builder)
+        where TActorService : ActorService
+        {
+            if (builder == null)
+                throw new ArgumentNullException(nameof(builder));
+
+            return builder.RegisterType<TActorService>().As<ActorService>();
```

Comment:
With the change above this would become `As<TActorService>` or `AsSelf` because we would be resolving the specific type. We should register the default `ActorService` in the `ServiceFabricModule` so that it can be used as the fallback.

Reply:
I see you have of course already registered the default `ActorService` in the `ServiceFabricModule` below. 👍